selfdrive/car/honda/hondacan.py

```diff
@@ -94,14 +100,15 @@ def create_ui_commands(pcm_speed, hud, civic, accord, idx):
     commands.append(make_can_msg(0x39f, msg_0x39f, idx, 0))
   return commands
 
-def create_radar_commands(v_ego, civic, accord, idx):
+def create_radar_commands(v_ego, civic, accord, crv, idx):
```

Comment:
Ugh yea, the honda interface code is the lowest quality. Would love a refactor pull request.